currency-converter/currency-con.js

```diff
@@ -7,13 +7,24 @@ const addTrackerTable = document.getElementById("addTrackedCur");
 
 const marketWatch = [];
 const alertArray = [];
-const openPriceArr = [];
+const openPriceArr = [];   // ex --> pushing  ["USD","EUR", 10]
 const trackingPriceTable = [];
 
 
+// Get updated rates form an external source  (using API)
+setInterval(() => {
+    fetch("https://api.currencyfreaks.com/v2.0/rates/latest?apikey=0c25017fdad747739d3a5f98c37cf903")
+    .then(dataRow => dataRow.json())
+    .then(data => {
+        marketWatch.length = 0;
+        marketWatch.push(data);
+        searchCurrency();
+    });
+},10000);
```

Comment:
If you make a call every 10 seconds, that's 360 calls per hour. Looking at the pricing for https://currencyfreaks.com/, you only get 1000 calls per month on the free plan. Since I assume you are on the free plan, you might want to dial this down or only update those rates when the application loads and then maybe let the user update manually by clicking a button.